README.md

```diff
@@ -146,6 +146,38 @@ The following example will generate Magento 2.2 documentation only.
 
 If you don't have the `_config.local.yml` file at the root of your `devdocs/` directory, the rake will generate all versions of the documentation.
 
+### To build v2.0 documentation
+
+Magento 2.0.18 was the final 2.0.x release. After March 2018, Magento 2.0.x stopped receiving security patches, quality fixes, and documentation updates.
+
+To ensure the quality of our public documentation for versions of Magento that are still supported and to avoid potential confusion about 2.0.x support, we removed all 2.0.x content from our public documentation websites on May 31, 2019.
+
+To build v2.0 documentation:
+
+> You can also use the GitHub interface to read the source Markdown files instead of building the site locally. Just navigate to the [`2.0`](https://github.com/magento/devdocs/tree/2.0) branch and click the file you want to view.
```

Comment:
Slight edit option for easier translating:

```suggestion
> You can also use the GitHub interface to read the source Markdown files instead of building the site locally. Navigate to the [`2.0`](https://github.com/magento/devdocs/tree/2.0) branch and click the file you want to view.
```